File: tasks/validTime.java
boolean validTime(String time) {

  class Helper {
  int charToInt(char symbol) {
    return symbol - '0';
  }
  }
  Helper h = new Helper();

  int hours = h.charToInt(time.charAt(0)) * 10 + h.charToInt(time.charAt(1)),
      minutes = h.charToInt(time.charAt(3)) * 10 + h.charToInt(time.charAt(4));

  if (hours < 24 && minutes < 60) {
    return true;
  }
  return false;
}

File: tasks/validTime.py
def validTime(time):

    def charToInt(symbol):
        return ord(symbol) - ord('0')

    hours = charToInt(time[0]) * 10 + charToInt(time[1])
    minutes = charToInt(time[3]) * 10 + charToInt(time[4])

    if hours < 24 and minutes < 60:
        return True
    return False
